src/style.css

```diff
@@ -1 +1,39 @@
-/* add styles here */
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+  padding-right: 5px;
+  width: 96px;
+  justify-content: space-between;
```

Comment:
Avoid fixing container sizes as per the checklist unless there's a specific requirement for it. Here, setting a fixed width for the `.stars` container might cause issues if the size of the stars changes or if additional stars are added. Consider using a more flexible layout that doesn't rely on a fixed width, or ensure there's a specific reason for this fixed width.